styles/Albumclient.js

```
@@ -120,6 +120,11 @@ const AlbumClient = () => {
     return [decodeURIComponent(name), random];
   }, [slug]);
 
+  // currentPath sẽ là: data/albumName
+  const currentPath = useMemo(() => {
+    return albumName ? `data/${albumName}` : '';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'currentPath'. Consider to delete it or rename it.

'currentPath' is defined here:


(lint/suspicious/noRedeclare)